ee/modules/600-flant-integration/hooks/pricing/envs_from_nodes.go

```diff
@@ -51,12 +51,13 @@ func ApplyPricingNodeFilter(obj *unstructured.Unstructured) (go_hook.FilterResul
 		n.Type = t
 	}
 
-	if _, ok := node.ObjectMeta.Labels["node-role.kubernetes.io/control-plane"]; !ok {
+	if nodeGroup, ok := node.ObjectMeta.Labels["node.deckhouse.io/group"]; !ok || nodeGroup != "master" {
 		return n, err
```

Comment:
Nit. It's better to be explicit. Not your change, so I don't insist on the change.

```suggestion
		return n, nil
```

Reply:
Doesn't seems right. It will return any non-master node without an error and it will be counter as master

Reply:
or I didn't get the clue

Reply:
It seems that non-master (or non-dedicated master) nodes expected to be returned in a same way as a master nodes, except with `n.MasterNodeInfo.IsDedicated` set to false.
If err isn't nil, function will exit on [line 44](https://github.com/deckhouse/deckhouse/blob/0bcfd48dce8562f02eef1feccb70e5a2062ddf55/ee/modules/600-flant-integration/hooks/pricing/envs_from_nodes.go#L44), so it's safe and desirable to apply this suggestion.